packages/minimongo/package.js

```diff
@@ -1,66 +1,46 @@
 Package.describe({
-  summary: "Meteor's client-side datastore: a port of MongoDB to Javascript",
-  version: '1.2.1'
+  summary: 'Meteor\'s client-side datastore: a port of MongoDB to Javascript',
+  version: '1.2.1',
 });
 
-Package.onUse(function (api) {
+Package.onUse(api => {
   api.export('LocalCollection');
   api.export('Minimongo');
+
   api.export('MinimongoTest', { testOnly: true });
   api.export('MinimongoError', { testOnly: true });
+
   api.use([
-    'underscore',
+    'diff-sequence', // This package is used to get diff results on arrays and objects
+    'ecmascript',
     'ejson',
+    'geojson-utils', // This package is used for geo-location queries such as $near
     'id-map',
-    'ordered-dict',
-    'tracker',
     'mongo-id',
+    'ordered-dict',
     'random',
-    'diff-sequence',
-    'ecmascript'
-  ]);
-  // This package is used for geo-location queries such as $near
-  api.use('geojson-utils');
-  // This package is used to get diff results on arrays and objects
-  api.use('diff-sequence');
-
-  api.addFiles([
-    'minimongo.js',
-    'wrap_transform.js',
-    'helpers.js',
-    'selector.js',
-    'sort.js',
-    'projection.js',
-    'modify.js',
-    'diff.js',
-    'id_map.js',
-    'observe.js',
-    'objectid.js'
+    'tracker',
   ]);
 
-  // Functionality used only by oplog tailing on the server side
-  api.addFiles([
-    'selector_projection.js',
-    'selector_modifier.js',
-    'sorter_projection.js'
-  ], 'server');
+  api.mainModule('main.js', 'client');
```

Comment:
Oh, sorry I didn't notice that. In that case, would you mind making the following changes:

- Create a new `minimongo_common.js` file that holds the current contents of `main.js`
- Rename `main.js` to `minimongo_client.js`, and just import/export the `minimongo_common.js` contents (and update the `package.js` to have `api.mainModule('minimongo_client.js', 'client');` so this is the new client mainline).
- Rename `main_server.js` to `minimongo_server.js` and import `minimongo_common.js` (and update the `package.js` accordingly)

This will then fall more in-line with client/server shared code approach used in other core packages. Thanks!